app/javascript/src/components/Dashboard/Contacts/ContactMenu.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+import { CONTACT_MENU_BAR_VALUES } from "./constants";
+
+const Menu = ({ showMenu }) => {
```

Comment:
- Go through the Slack thread and understand why redundant namespacing is not good.
- Also, the React component name should be exactly same as the file name.

Reply:
Ok, I will replace the files and the components with proper naming.